lib/reducers/adapterReducer.js

```diff
@@ -371,6 +378,14 @@ function serverSetupApplied(state) {
     return state.setIn(['selectedAdapter', 'isServerSetupApplied'], true);
 }
 
+function showConParamsDialog(state, bool) {
```

Comment:
Also here, I think it's worthwhile to use the full name, i.e. `showConnectionParamsDialog`.